vmp/helper/collections.go

```diff
@@ -2,68 +2,54 @@
 
 package helper
 
-// Filter returns a new slice containing all items in the slice that satisfy the predicate f.
-func Filter(vs []interface{}, f func(interface{}) bool) []interface{} {
-	vsf := make([]interface{}, 0)
-	for _, v := range vs {
-		if f(v) {
-			vsf = append(vsf, v)
+// InterfaceSliceEqual tells whether a and b contain the same elements.
+// A nil argument is equivalent to an empty slice.
+func InterfaceSliceEqual(a, b []interface{}) bool {
```

Comment:
Please add "Is" prefix for readability. IsInterfaceSliceEqual or something.

Reply:
Good idea, will do